tests/apps/actorjava/Dockerfile-windows

```diff
@@ -0,0 +1,31 @@
+# escape=`
+ARG JAVA_VERSION=11.0.6
+
+FROM mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019 as build
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV chocolateyUseWindowsCompression=false
+RUN iex (wget 'https://chocolatey.org/install.ps1' -UseBasicParsing)
+
+RUN choco feature enable -n=allowGlobalConfirmation
+RUN choco feature disable -n=showDownloadProgress
+RUN choco config set cachelocation C:\chococache
+
+RUN choco install ojdkbuild11 maven --limit-output --timeout 3600; Remove-Item C:\chococache  -Recurse -Force
+
+RUN mkdir "c:\build"
+WORKDIR "c:\build"
```

Comment:
Recommend you do this part separately from the build process. We actually have a workflow to build and upload containers that we don't want to rebuild from scratch every time:

https://github.com/dapr/dapr/blob/master/.github/workflows/dapr-base-containers.yaml

If you make a container with maven already installed, this can really cut down on build times. Also, idk what maven/jdk depend on, but if you can get away with using the windows nanoserver container, that can save you a lot of container download time. (It does look like there are nanoserver openjdk prebuilt containers https://hub.docker.com/_/openjdk/?tab=tags&page=1&ordering=last_updated&name=nanoser )